app/views/cash_account_v2.scala.html

```diff
@@ -51,7 +51,11 @@
 
     @viewModel.cashAccountBalance
 
-    @viewModel.cashStatementNotification.map(identity)
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
```

Comment:
Could you please add a test to check this style class as it is has been an important requirement here? If takes time then leave it.